src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
>There was always repo-specific code for the native builds, but the stuff that could have been same was actually same.

You are correct; I should not have said exact same. Although I directly copied the CoreCLR native build logic into CoreFX, I did have to make a few modifications that were CoreFX specific.

What I should have said was that there was enough shared code between CoreCLR and CoreFX at the inception of the CoreFX native windows build that we could have relatively easily combined the logic into one set without adding too much complexity.